src/component/table/Table-test.js

```diff
@@ -0,0 +1,72 @@
+import test from 'ava';
```

Comment:
This file is throwing a bunch of eslint errors. Need to either be fixed or ignored.